classes/Util/class.xvmpVideoPlayer.php

```diff
@@ -62,28 +62,31 @@ public function __construct($video, $embed = false) {
 		$this->embed = $embed;
 	}
 
-    /**
-     * @param $load_observer
-     */
-    public static function loadVideoJSAndCSS($load_observer) {
+
+	/**
+	 * @param $load_observer
+	 */
+	public static function loadVideoJSAndCSS($load_observer) {
 		$tpl = $GLOBALS['tpl']; // necessary because of LM Bug
 		if ($load_observer) {
 			$tpl->addJavaScript(ilViMPPlugin::getInstance()->getDirectory() . '/js/xvmp_observer.js');
-
 		}
 		$tpl->addCss(ilViMPPlugin::getInstance()->getDirectory() . '/templates/default/video.css');
 
-                $tpl->addJavaScript(ilViMPPlugin::getInstance()->getDirectory() . '/vendor/video-js-7.5.4/video.js');
-                $tpl->addJavaScript(ilViMPPlugin::getInstance()->getDirectory() . '/vendor/video-js-7.5.4/videojs-contrib-quality-levels.js');
-                $tpl->addJavaScript(ilViMPPlugin::getInstance()->getDirectory() . '/vendor/video-js-7.5.4/videojs-hls-quality-selector.js');
-                $tpl->addCss(ilViMPPlugin::getInstance()->getDirectory() . '/vendor/video-js-7.5.4/video-js.css');
+		$tpl->addJavaScript(ilViMPPlugin::getInstance()->getDirectory() . '/node_modules/video.js/dist/video.min.js');
+		$tpl->addCss(ilViMPPlugin::getInstance()->getDirectory() . '/node_modules/video.js/dist/video-js.min.css');
+		$tpl->addJavaScript(ilViMPPlugin::getInstance()->getDirectory()
+			. '/node_modules/videojs-contrib-quality-levels/videojs-contrib-quality-levels.min.js');
+		$tpl->addJavaScript(ilViMPPlugin::getInstance()->getDirectory()
+			. '/node_modules/videojs-hls-quality-selector/videojs-hls-quality-selector.min.js');
```

Comment:
this should also contain the dist directory: . '/node_modules/videojs-hls-quality-selector/dist/videojs-hls-quality-selector.min.js');